swagger.yaml

```diff
@@ -6204,6 +6252,11 @@ definitions:
     type: object
     description: List of IP address pools for services.
```

Comment:
Nit: we store here a single subnet, not a list of subnets. Maybe, to copy-paste from above, "IP address block for service IP blocks" ?